docs/source/references.bib

```diff
@@ -192,6 +205,15 @@ @Article {daubechies-2004-iterative
   doi =		 {10.1002/cpa.20042}
 }
 
+@Misc {diamond-2018-odp,
+  author =	 {Steven Diamond and Vincent Sitzmann and Felix Heide
+                  and Gordon Wetzstein},
+  title =	 {Unrolled Optimization with Deep Priors},
+  year =	 2018,
+  eprint =	 {arXiv:1705.08041v2},
```

Comment:
On arXiv in 2018, but still not published anywhere?

Reply:
Yes, still not published anywhere, but referenced everywhere.